src/keysignature.ts

```diff
@@ -240,9 +246,16 @@ export class KeySignature extends StaveModifier {
     }
   }
 
-  // Format and position the modifier
+  /**
+   *  Format and position the modifier.
```

Comment:
Sorry I missed this in my previous review. There is an extra space between \* and Format.

Reply:
done